roles/install/vars/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+__default_pkgs:
+  - lemonldap-ng
+  - lemonldap-ng-doc
+
+__redhat_apache_pkgs:
+  - httpd
+  - mod_perl
+  - mod_fcgid
```

Comment:
mod_perl and mod_fcgid should be pulled in by the rpm packages dependencies.

Reply:
removed!